### ISSUE TITLE
CI: Nightly: Update Cilium from v1.2 to master

### ISSUE THREAD

**Issue by brb** [cilium/cilium#7187]
Observed in https://jenkins.cilium.io/job/cilium-master-Nightly/108/. Unfortunately, no dumps are available, only this log snippet:

```
00:14:22 STEP: Installing a cleaning state of Cilium
00:14:22 STEP: Installing kube-dns
00:14:22 STEP: Deploying etcd-operator
00:14:22 STEP: Cilium "v1.2" is installed and running
00:24:26 === Test Finished at 2019-02-21T23:24:22Z====
00:24:26 ===================== TEST FAILED =====================
00:24:26 cmd: kubectl get pods -o wide --all-namespaces
00:24:26 Exitcode: 0 
00:24:26 Stdout:
00:24:26  	 NAMESPACE     NAME                                    READY   STATUS    RESTARTS   AGE     IP              NODE     NOMINATED NODE
00:24:26 	 kube-system   cilium-etcd-operator-77b9b44d4c-zp5kr   1/1     Running   0          10m     192.168.36.13   k8s3     <none>
00:24:26 	 kube-system   coredns-779ffd89bd-w97m8                0/1     Pending   0          10m     <none>          <none>   <none>
00:24:26 	 kube-system   etcd-k8s1                               1/1     Running   0          14m     192.168.36.11   k8s1     <none>
00:24:26 	 kube-system   etcd-operator-86bff97c4f-878xz          0/1     Pending   0          4m55s   <none>          <none>   <none>
00:24:26 	 kube-system   kube-apiserver-k8s1                     1/1     Running   0          14m     192.168.36.11   k8s1     <none>
00:24:26 	 kube-system   kube-controller-manager-k8s1            1/1     Running   0          13m     192.168.36.11   k8s1     <none>
00:24:26 	 kube-system   kube-proxy-bzbmq                        1/1     Running   0          12m     192.168.36.12   k8s2     <none>
00:24:26 	 kube-system   kube-proxy-sspsz                        1/1     Running   0          14m     192.168.36.11   k8s1     <none>
00:24:26 	 kube-system   kube-proxy-sxl59                        1/1     Running   0          10m     192.168.36.14   k8s4     <none>
00:24:26 	 kube-system   kube-proxy-tqtxg                        1/1     Running   0          11m     192.168.36.13   k8s3     <none>
00:24:26 	 kube-system   kube-scheduler-k8s1                     1/1     Running   0          14m     192.168.36.11   k8s1     <none>
00:24:26 	 
00:24:26 Stderr:
00:24:26  	 
00:24:26 
00:24:32 ===================== Existing AfterFailed =====================
00:25:12 <Checks>
00:25:12 Number of 'context deadline exceeded' in logs: 0
00:25:12 Number of 'level=error' in logs: 0
00:25:12 Number of 'level=warning' in logs: 0
00:25:12 Number of 'Cilium API handler panicked' in logs: 0
00:25:12 Number of 'Goroutine took lock for more than' in logs: 0
00:25:12 Cilium pods: []
00:25:12 Netpols loaded: 
00:25:12 CiliumNetworkPolicies loaded: 
00:25:12 Endpoint Policy Enforcement:
00:25:12 Pod   Ingress   Egress
00:25:12 
00:25:12 </Checks>
00:25:12 
00:25:12 [[ATTACHMENT|57a18273_NightlyExamples_Upgrade_test_Update_Cilium_from_v1.2_to_master.zip]]
00:25:12 
00:25:12 • Failure [1054.552 seconds]
00:25:12 NightlyExamples
00:25:12 /home/jenkins/workspace/cilium-master-Nightly/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:377
00:25:12   Upgrade test
00:25:12   /home/jenkins/workspace/cilium-master-Nightly/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:377
00:25:12     Update Cilium from v1.2 to master [It]
00:25:12     /home/jenkins/workspace/cilium-master-Nightly/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:409
00:25:12 
00:25:12     cilium was not able to get into ready state
00:25:12     Expected
00:25:12         <*errors.errorString | 0xc42044cd80>: {
00:25:12             s: "Timeout reached: timed out waiting for pods with filter -l k8s-app=cilium to be ready",
00:25:12         }
00:25:12     to be nil
00:25:12 
00:25:12     /home/jenkins/workspace/cilium-master-Nightly/src/github.com/cilium/cilium/test/k8sT/Updates.go:159
```

**Comment by aanm** [comment]
@brb v1.2 is EOF so we no longer support comparability on upgrades to master. This can easily be fixed by removing the combination of v1.2 -> master in the test upgrade for nightly builds.

**Comment by brb** [comment]
@aanm Thanks, I've filed a PR ^^ for this.
